package.json

```diff
@@ -1,191 +1,21 @@
 {
-  "name": "aurelia-framework",
-  "version": "1.1.5",
-  "description": "The aurelia framework brings together all the required core aurelia libraries into a ready-to-go application-building platform.",
-  "keywords": [
-    "aurelia",
-    "framework",
-    "spa"
-  ],
+	"name": "aurelia-ux",
```

Comment:
Seems like the name, bugs, etc. aren't quite right

Reply:
Copy paste got me.